README.md

```diff
@@ -1,89 +1,59 @@
-# Docker Images from Oracle
+# Installing Oracle DB,  APEX and ORDS in Docker
 
-This repository contains [Dockerfiles](https://docs.docker.com/engine/reference/builder/)
-and samples to build [Docker](https://www.docker.com/what-docker) images for
-Oracle commercial products and [Oracle sponsored open source projects](https://opensource.oracle.com).
 
-## Container Images on GitHub
+## Step 1: Installing Docker
 
-These images will require you to download any required Oracle commercial
-software before installation. If you want commercial software downloaded for you,
- view [Pre-Built Images with Commercial Software](#pre-built-images-with-commercial-software).
+Follow the instructions on the Docker official documentation to install Docker Desktop on Windows:
+- [Docker Desktop Installation for Windows](https://docs.docker.com/desktop/install/windows-install/)
```

Comment:
You should provide instructions for how to deploy this on Oracle Linux using Podman, as this would be the most common deployment scenario on an OCI instance.